i/immutable-js/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "prachikurade1",
+    "package_name": "immutable-js",
+    "github_url": "https://github.com/immutable-js/immutable-js",
+    "version": "v4.1.0",
```

Comment:
I see latest release version is [v4.3.6](https://github.com/immutable-js/immutable-js/releases/tag/v4.3.6).
SO I will suggest we add support for latest version by default.